CHANGELOG.md

```diff
@@ -5,21 +5,38 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/USGS-WiM/stnweb2/tree/dev)
+## [Unreleased](link to development branch)
 
-### Added
+### Added 
 
 -
 
-### Changed
+### Changed  
 
 -
 
-### Removed
+### Deprecated 
 
 -
 
-### Fixed
+### Removed 
+
+- 
+
+### Fixed  
+
+- 
+
+### Security  
 
 - 
 
+## [v0.0.1](link to release) - 2022-01-17
```

Comment:
Change the link to a real link so the formatting looks better and people know what it will look like: https://github.com/USGS-WiM/projectName/releases/tag/v1.0.0